app/utility/ls_date.rb

```diff
@@ -12,6 +12,12 @@
 class LsDate
   include Comparable
   attr_reader :date_string, :specificity, :year, :month, :day
+
+  BASIC_FORMAT = "%B %e, %Y, %l%p".freeze
+
+  def self.pretty_print(date)
```

Comment:
minor quibble here:

LsDate is a custom date class, but ` LsDate.pretty_print ` takes a standard datetime not an LsDate. 

I know i was the one who suggested that you put that here 🦉 ! Perhaps we could add a comment or a type-checking exception:
``` rb
# input: <DateTime> ( not LsDate)
def self.pretty_print
  raise ArgumentError, "Invalid input type" unless date.is_a? DateTime
 date.strftime(BASIC_FORMAT) 
end

Reply:
sounds fine. happy to put it any place that might be a convenient "generic date" util holder. but this seems like the most logical available place ATM. i like the type guard solution.